executor/executor.go

```diff
@@ -1111,6 +1111,125 @@ func (e *AggregationExec) innerNext() (ret bool, err error) {
 	return true, nil
 }
 
+// StreamAggExec deals with all the aggregate functions.
+// It is built from Aggregate Plan. When Next() is called, it reads all the data from Src and updates all the items in AggFuncs.
+type StreamAggExec struct {
+	Src                Executor
+	schema             expression.Schema
+	ResultFields       []*ast.ResultField
+	executed           bool
+	hasData            bool
+	ctx                context.Context
+	AggFuncs           []expression.AggregationFunction
+	GroupByItems       []expression.Expression
+	curGroupEncodedKey []byte
+	curGroupKey        []types.Datum
+	tmpGroupKeyBuffer  []types.Datum
```

Comment:
`tmpGroupKeyBuffer` -> `tmpGroupKey`, as `buffer` is usually used in bytes type.